README.md

```diff
@@ -216,37 +215,37 @@ A file extension is not required for `content.url`.  A content's [tile format](#
+3D Tiles uses a right-handed Cartesian coordinate system; that is, the cross product of _x_ and _y_ yields _z_. 3D Tiles defines the _z_ axis as up for local Cartesian coordinate systems (see the [Tile transform](#tile-transform) section).  A tileset's global coordinate system will often be [WGS84 coordinates](http://earth-info.nga.mil/GandG/publications/tr8350.2/wgs84fin.pdf), but it doesn't have to be, e.g., a power plant may be defined fully in its local coordinate system for use with a modeling tool without a geospatial context.
 
-`b3dm` and `i3dm` tiles embed glTF. According to the [glTF spec](https://github.com/KhronosGroup/glTF/tree/master/specification/1.0#coordinate-system-and-units) glTF uses a right-handed coordinate system and defines the y axis as up. By default embedded models are considered to be y-up, but in order to support a variety of source data, including models defined directly in WGS84 coordinates, embedded glTF models may be defined as x-up, y-up, or z-up with the `asset.gltfUpAxis` property of `tileset.json`. In general an implementation should transform glTF assets to z-up at runtime to be consistent with the z-up coordinate system of the bounding volume hierarchy.
+`b3dm` and `i3dm` tiles embed glTF. According to the [glTF spec](https://github.com/KhronosGroup/glTF/tree/master/specification/1.0#coordinate-system-and-units), glTF uses a right-handed coordinate system and defines the _y_ axis as up. By default, embedded models are considered to be _y_-up, but in order to support a variety of source data, including models defined directly in WGS84 coordinates, embedded glTF models may be defined as _x_-up, _y_-up, or _z_-up with the `asset.gltfUpAxis` property of tileset.json. In general, an implementation should transform glTF assets to _z_-up at runtime to be consistent with the _z_-up coordinate system of the bounding volume hierarchy.
 
-The units for all linear distances are meters.
+The unit for all linear distances is meters.
 
 All angles are in radians.
 
-3D Tiles do not explicitly store Cartographic coordinates (longitude, latitude, and height); these values are implicit in WGS84 Cartesian coordinates, which are efficient for the GPU to render since they do not require a non-affine coordinate transformation.  A 3D Tiles tileset can include application-specific metadata, such as Cartographic coordinates, but the semantics are not part of the 3D Tiles specification.
+3D Tiles does not explicitly store cartographic coordinates (longitude, latitude, and height); these values are implicit in WGS84 Cartesian coordinates, which are efficient for the GPU to render since they do not require a non-affine coordinate transformation.  A 3D Tiles tileset can include application-specific metadata, such as cartographic coordinates, but the semantics are not part of the 3D Tiles specification.
```

Comment:
Should we add to the [style guide](https://github.com/AnalyticalGraphicsInc/agi-cesium-people/tree/master/StyleGuide) that "3D Tiles" should be treated as singular?

Reply:
Yes, I have a couple of things to add since going through this. Thanks for the reminder!